src/modules/shop/shop.controller.ts

```diff
@@ -322,33 +325,74 @@ export class ShopController {
   public postOrder = async (req: TRequest<OrderDto>, res: TResponse, next: NextFunction) => {
     const { cartId } = req.dto;
     const userId = req.user.id;
+    const lineItems = [];
+
+    const stripe = new Stripe(env.stripePrivate);
 
     try {
       const cartItems = await CartItem.findAll({ where: { cartId: cartId }, attributes: ["productId", "quantity"] });
 
+      for (const item of cartItems) {
+        const productId = item.dataValues.productId;
+
+        const product = await Product.findOne({ attributes: ["title", "price"], where: { id: productId } });
```

Comment:
Always add exception handling when you are querying to database or if you are making an API call